<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>MainWindowClass</class>
  <widget class="QMainWindow" name="MainWindowClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>400</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralWidget"/>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>21</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="actionSaveImage"/>
        <addaction name="separator"/>
        <addaction name="actionExit"/>
      </widget>
      <widget class="QMenu" name="menuL_System">
        <property name="title">
          <string>L-System</string>
        </property>
        <addaction name="actionRandomGeneration"/>
        <addaction name="actionGreedyInverse"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuL_System"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="actionExit">
      <property name="text">
        <string>Exit</string>
      </property>
    </action>
    <action name="actionRandomGeneration">
      <property name="text">
        <string>Random Generation</string>
      </property>
    </action>
    <action name="actionGreedyInverse">
      <property name="text">
        <string>Greedy Inverse</string>
      </property>
    </action>
    <action name="actionSaveImage">
      <property name="text">
        <string>Save Image</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="MainWindow.qrc"/>
  </resources>
  <connections/>
</ui>
